nix/default.nix

```diff
@@ -145,6 +145,7 @@ in rec {
       #!${pkgs.bash}/bin/bash
       # Set the JAVA_HOME to our JDK
       export JAVA_HOME=${jdk.home}
+      unset LD_LIBRARY_PATH
```

Comment:
What's the reason for this?

Reply:
I need this change locally to be able to start bazel from Intellij IDEA. Removed.